bin/new_plugin_repo.rb

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH << File.expand_path("../lib", __dir__)
+
+require 'bundler/setup'
+require 'manageiq/release'
+require 'optimist'
+
+opts = Optimist.options do
+  opt :repo, "The repo to update.", :required => true, :type => :string
+
+  opt :dry_run, "", :default => false
+end
+
+repo = ManageIQ::Release.repo_for(opts[:repo])
+labels = ManageIQ::Release::Labels[repo.github_repo]
+unless repo && labels
+  STDERR.puts "ERROR: First update config/repos.yml and config/labels.yml with the new repo"
+  exit 1
+end
+
+puts "\n** Updating Repo Settings"
+ManageIQ::Release::UpdateRepoSettings.new(repo.github_repo, opts.slice(:dry_run)).run
+puts "\n** Updating Branch Protection"
+ManageIQ::Release::UpdateBranchProtection.new(repo.github_repo, opts.slice(:dry_run).merge(:branch => "master")).run
+puts "\n** Updating Labels"
+ManageIQ::Release::UpdateLabels.new(repo.github_repo, opts.slice(:dry_run)).run
+
+puts "\n** Preparing Pull Request"
+ManageIQ::Release::PullRequestBlasterOuter.new(repo, opts.slice(:dry_run).merge(
```

Comment:
Totally unrelated to this but BlasterOuter with a single repo seems strange, would have thought e.g. `ManageIQ::Release::PullRequest` would take a single repo and `PullRequestBlasterOuter` would take an array

Reply:
Yeha, I'm considering renaming that class...it only blasts to one repo (like every other class)